edb/server/pgcon/connect.py

```diff
@@ -104,11 +109,17 @@ def _build_init_con_script(*, check_pg_is_in_recovery: bool) -> bytes:
         {SETUP_CONFIG_CACHE_SCRIPT}
         {SETUP_DML_DUMMY_TABLE_SCRIPT}
 
+        CREATE CONSTRAINT TRIGGER _edgecon_state_local_reset
+        AFTER INSERT ON _edgecon_state
+        DEFERRABLE INITIALLY DEFERRED
+        FOR EACH ROW EXECUTE FUNCTION edgedb._clear_fe_local_sql_settings();
```

Comment:
Ah, this will be tricky for `edb.pgsql.patches` to patch up, let me think of a workaround

Reply:
Fixed, see commit below.